gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -141,10 +141,6 @@ class GateConfig extends RedisHttpSessionConfiguration {
   @Autowired
   ServiceConfiguration serviceConfiguration
 
-  /**
-   * This needs to be before the yaml converter in order for json to be the default
-   * response type.
-   */
   @Bean
```

Comment:
we should be able to delete this whole bean!

Reply:
oh, cool, let me do that